requirements.txt

```diff
@@ -1,15 +0,0 @@
-Django<2.1
```

Comment:
The Dockerfiles for concordia and importer images reference these requirements.txt files. If they're going to be deleted, the Dockerfiles need to be updated. So does the README, probably, with developer environment setup instructions. We could use instruction on how to keep the imports correctly ordered going forward.

Reply:
I hadn't gotten to the Docker files yet – the hope was that we'd use this in both places since I really like the fixity checks on dependencies.